browser/help/topics/combined-freq-stats.md

```diff
@@ -19,10 +19,19 @@ The expected number of variants to reach this threshold by chance is 9,100 (out
 
 ![CMH p-value distribution](cmh-pval.png)
 
+Example [variant](http://34.36.169.116/variant/1-55039847-G-A?dataset=gnomad_r4) flagged based on CMH test:
+![variant flagged with CMH test](variant-cmh-flag.png)
+
+Example [variant](http://34.36.169.116/variant/1-55052214-G-T?dataset=gnomad_r4) flagged based on contingency table test:
+![variant flagged with contingency table test](variant-ctt-flag.png)
+
 ### Why have we added these statistical tests
 
 For the first time, in gnomAD v4.0, we released a combined filtering allele frequency (FAF), integrating variant frequencies across the 734,947 exomes and 76,215 genomes. This integration brings the advantage of a larger, more diverse sample set but also introduces challenges. These challenges stem from differences in sequencing and processing methodologies, as well as variations in sample composition due to ascertainment biases. Addressing these challenges is crucial for providing accurate and reliable genetic insights.
 
 These enhancements and warnings are part of our ongoing commitment to data integrity and usability in gnomAD. By identifying variants with significant frequency discrepancies and providing contextual warnings, we aim to equip researchers with the knowledge needed to make informed decisions in their genetic analyses. We encourage users to explore these new statistical analyses as they navigate gnomAD v4.1.
 
-For more information about these flags and how to approach interpreting frequencies at the flagged loci, please see our [blog post](https://gnomad.broadinstitute.org/news/2024-04-gnomad-v4-1).
+For more information about this flag and how to approach interpreting frequencies at the flagged loci, please see our [blog post](https://gnomad.broadinstitute.org/news/2024-04-gnomad-v4-1).
+
+### Discrepant frequency MVP
+Note that the statistical tests and associated discrepant frequency flag added in gnomAD v4.1 are a minimum viable product (MVP) version. We will continue to iterate and improve on these analyses in future gnomAD releases.
```

Comment:
This is good, but I think we also need something to warn users. Something like: "Users should exercise caution when using these results for interpretation, as they are preliminary and subject to revisions in future updates."